packages/react-native-reanimated/src/commonTypes.ts

```diff
@@ -5,6 +5,70 @@ import type {
   TransformsStyle,
   ImageStyle,
 } from 'react-native';
+import type { WorkletRuntime } from './runtimes';
+
+export interface IReanimatedModule {
+  registerSensor(
+    sensorType: number,
+    interval: number,
+    iosReferenceFrame: number,
+    handler: ShareableRef<(data: Value3D | ValueRotation) => void>
+  ): number;
+  unregisterSensor(sensorId: number): void;
+  registerEventHandler<TValue>(
+    eventHandler: ShareableRef<TValue>,
+    eventName: string,
+    emitterReactTag: number
+  ): number;
+  unregisterEventHandler(id: number): void;
```

Comment:
Let's add newlines in between for consistency.
```suggestion
  registerSensor(
    sensorType: number,
    interval: number,
    iosReferenceFrame: number,
    handler: ShareableRef<(data: Value3D | ValueRotation) => void>
  ): number;

  unregisterSensor(sensorId: number): void;

  registerEventHandler<TValue>(
    eventHandler: ShareableRef<TValue>,
    eventName: string,
    emitterReactTag: number
  ): number;

  unregisterEventHandler(id: number): void;
```